### PR TITLE
Fix request semantics of fty-msgbus-cli

### PR DESCRIPTION
**Pull request by boricj**

### CODE REVIEW

**Review by perrettecl — APPROVED** [review]

File: src/fty-msgbus-cli.cc [hunk 1]
```diff
@@ -165,17 +165,12 @@ void sendRequestReply(messagebus::MessageBus* msgbus, int argc, char** argv) {
...
-        std::raise(SIGINT);
-    });
-
-    // Wait until interrupt.
-    std::unique_lock<std::mutex> lock(g_mutex);
-    g_cv.wait(lock, [] { return g_exit; });
+    try {
+        dumpMessage(msgbus->request(queue, msg, stoi(timeout)));
+    }
+    catch (std::exception& ex) {
```
Comment (etnKlendathu) [inline src/fty-msgbus-cli.cc]: picky note: const std::exception :)

File: src/fty-msgbus-cli.cc [hunk 2]
```diff
@@ -165,17 +165,12 @@ void sendRequestReply(messagebus::MessageBus* msgbus, int argc, char** argv) {
...
-    msgbus->receive(clientName, [](messagebus::Message msg) { 
-        dumpMessage(msg);
-        std::raise(SIGINT);
-    });
-
-    // Wait until interrupt.
-    std::unique_lock<std::mutex> lock(g_mutex);
-    g_cv.wait(lock, [] { return g_exit; });
+    try {
+        dumpMessage(msgbus->request(queue, msg, stoi(timeout)));
```
Comment (etnKlendathu) [inline src/fty-msgbus-cli.cc:169]: picky note: timeout is string user input... stoi without check is just dangerous to hang messagebus.

Comment (boricj) [reply src/fty-msgbus-cli.cc:169]: We assume that the developers which use this tool know what they are doing. Passing a timeout that lasts until the heat death of the universe is really the least of our concerns when someone has access to the Malamute bus.

Comment (etnKlendathu) [reply src/fty-msgbus-cli.cc:169]: :) perfect answer :)
Picky note, i mean it's ok... just very minor thing... up to you to fix it.

**Review by etnKlendathu — APPROVED** [review]

**Review by stark-dev — APPROVED** [review]

**Pull request merged**